src/styles/icons.css

```diff
@@ -16,60 +28,36 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-involved:before {
-  content: "\e906";
+.icon-phone:before {
+  content: "\e900";
 }
-.icon-family-large:before {
+.icon-arrow-right:before {
   content: "\e901";
 }
-.icon-pound:before {
+.icon-users:before {
   content: "\e902";
 }
-.icon-police-hat-large:before {
+.icon-email:before {
   content: "\e903";
 }
-.icon-leaving-large:before {
+.icon-hospital:before {
   content: "\e904";
 }
-.icon-pound-note-large:before {
+.icon-question:before {
   content: "\e905";
 }
-.icon-user:before {
-  content: "\f007";
-}
 .icon-book:before {
-  content: "\f02d";
-}
-.icon-pencil:before {
-  content: "\f040";
-}
-.icon-check-square-o:before {
-  content: "\f046";
-}
-.icon-info-circle:before {
-  content: "\f05a";
-}
-.icon-arrow-left:before {
-  content: "\f060";
-}
-.icon-arrow-right:before {
-  content: "\f061";
-}
-.icon-phone:before {
-  content: "\f095";
-}
-.icon-group:before {
-  content: "\f0c0";
+  content: "\e906";
 }
-.icon-users:before {
-  content: "\f0c0";
+.icon-info:before {
+  content: "\e907";
 }
-.icon-envelope:before {
-  content: "\f0e0";
+.icon-envelop:before {
```

Comment:
Is the mis-spell delibrate?

Reply:
yes, that's how the file was generated. there are two options: one is icon-email the other icon-envelop. I remove the latter as i did not use it.